src/applications/edu-benefits/0993/containers/ConfirmationPage.jsx

```diff
@@ -86,9 +85,7 @@ export class ConfirmationPage extends React.Component {
           )}
         </div>
         <div>
-          <AskVAQuestions>
-            <GetFormHelp />
-          </AskVAQuestions>
+          <FormFooter formConfig={{ getHelp: GetFormHelp }} />
```

Comment:
Was the edu GetFormHelp, like the hca gethelp component, also updated? If so, should it be included in this PR?

Reply:
We had updated it prior to launch, so I don't think I changed it here.